cpp/cmake/thirdparty/CUDF_GetArrow.cmake

```diff
@@ -123,4 +124,4 @@ endfunction()
 
 set(CUDF_VERSION_Arrow 1.0.1)
 
-find_and_configure_arrow(${CUDF_VERSION_Arrow} ${CUDF_USE_ARROW_STATIC})
+find_and_configure_arrow(${CUDF_VERSION_Arrow} ${CUDF_USE_ARROW_STATIC})
```

Comment:
Missing a newline at the end of the file.

Reply:
updated in latest commit